### PR TITLE
fix(#1112): fix multi parameters for multilist component

### PR DESCRIPTION
**Pull request by MrCode1997**
- [x] That change is need bracuse of bug that change the urlparamters in the URL when the props :URLParams is set to "true" on MultiList component. So with this change we are able to use MultiList component with url params with no bug.

Thanks All.

### CODE REVIEW

**Comment by bietkul** [comment]
@MrCode1997 I'm not sure what're you doing here, Can you please first explain the issue with a reproducible example? A code sandbox link will be much helpful.
Thanks

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
 	"private": true,
 	"name": "reactivesearch-packages",
+	"version": "1.0.0",
```
Comment (bietkul) [inline package.json:4]: @MrCode1997 We don't need this change.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (MrCode1997) [reply package.json:4]: @bietkul Ok, perfect probably was a refuse. i can remove it.

**Comment by MrCode1997** [comment]
> @MrCode1997 I'm not sure what're you doing here, Can you please first explain the issue with a reproducible example? A code sandbox link will be much helpful.
> Thanks

Hi @bietkul there is an open bug: https://github.com/appbaseio/reactivesearch/issues/1112
So i solved this bug. This is the only thing that i do.

**Comment by gpassarelli** [comment]
Hi Guys, this PR solded the issue #1112 for any, can you guys by any change approve this PR?

**Comment by bietkul** [comment]
This issue has been fixed in https://github.com/appbaseio/reactivesearch/pull/1602

**Pull request closed without merging**